app/workers/education_form/process10203_submissions.rb

```diff
@@ -150,10 +150,13 @@ def repeat_form?(unprocessed_form, processed_form)
     # Set status to DENIED when isPursuingTeachingCert in form data is 'no' (false)
     #   and isEnrolledStem is 'no' (false)
     #   or EVSS data for a user shows there is more than 6 months of remaining_entitlement
+    #
+    # This is only checking EVSS data until form questions that affect setting to DENIED have been reviewed
     def process_submission(submission, gi_bill_status, user_has_poa)
-      submission_form = format_application(submission)
-      status = if (!submission_form.enrolled_stem && !submission_form.pursuing_teaching_cert) ||
-                  more_than_six_months?(gi_bill_status)
+      # submission_form = format_application(submission)
+      # status = if (!submission_form.enrolled_stem && !submission_form.pursuing_teaching_cert) ||
```

Comment:
Is there a reason to leave the commented out code? git history is pretty nice for keeping track of things like this in the case it's needed again in the future.

Reply:
These tests will be added back once the language of `submission_form.enrolled_stem` and `submission_form.pursuing_teaching_cert` questions in 10203 is reviewed and if additional form questions are added to logic.

Used commenting so that when questions are allowed to be part of determining if 10203 is denied can easily uncomment and test.

Reply:
removed